bin/cheritest/Makefile.inc

```diff
@@ -0,0 +1,3 @@
+# $FreeBSD$
+
+.include "${.PARSEDIR}/../Makefile.inc"
```

Comment:
```suggestion
```
Do we need this? ../Makefile.inc is automatically included when you use bsd.prog.mk.

Reply:
Yes, because `${.CURDIR}` points at the subdirectories inside `bin/cheritest` and so `${.CURDIR}/../Makefile.inc` is this file, meaning they won't include `bin/Makefile.inc` ie `${.CURDIR}/../../Makefile.inc` otherwise.